.run/query-coordinator.run.xml

```diff
@@ -0,0 +1,18 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Is this an intellij run configuration file? My instinct would be to add this path to .gitignore rather than checking it in.

Reply:
👍 That's reasonable. Anyone washing to run the service from IntelliJ will want it. I'm ok w/ you checking it in.

Reply:
Also note that this uses a macro variable $ProjectFileDir$ and will work on anyones machine.